packages/web/src/components/pages/Tests/TestsList/TestsList.tsx

```diff
@@ -65,49 +68,35 @@ const TestsList: FC = () => {
     dataTestModalRoot: 'add-a-new-test-modal',
   });
 
-  const [abortAll] = useAbortAllTestExecutionsMutation();
   const onItemClick = useDashboardNavigate((item: Test) => `/tests/${item.name}`);
-  const onItemAbort = useCallback((item: Test) => {
-    abortAll({id: item.name})
-      .unwrap()
-      .catch(() => {
-        notificationCall('failed', 'Something went wrong during test execution abortion');
-      });
-  }, []);
 
   if (error) {
     return <Error title={(error as any)?.data?.title} description={(error as any)?.data?.detail} />;
   }
 
-  if (entityPromoComponent) {
-    return (
-      <>
-        {isValidElement(entityPromoComponent)
-          ? cloneElement(entityPromoComponent, {list: 'tests'} as Partial<unknown>)
-          : null}
-      </>
-    );
+  if (EntityPromoComponent) {
+    return <EntityPromoComponent list="tests" />;
   }
 
   return (
-    <EntityListContent
+    <EntityView
       itemKey="test.name"
       CardComponent={TestCard}
       onItemClick={onItemClick}
-      onItemAbort={onItemAbort}
       entity="tests"
       pageTitle="Tests"
       addEntityButtonText="Add a new test"
-      pageDescription={PageDescription}
+      pageDescription={<PageDescription />}
       emptyDataComponent={EmptyTests}
       initialFiltersState={initialFilters}
       dataTest="add-a-new-test-btn"
       queryFilters={filters}
       setQueryFilters={setFilters}
-      data={tests}
-      isLoading={isLoading}
+      data={tests ?? []}
```

Comment:
Better to allow undefined - the `[]` passed here will always have a new identity, so it will always call `useEffect` that has it as a dependency:

```suggestion
      data={tests}
```